grass7/temporal/.flake8

```diff
@@ -0,0 +1,46 @@
+[flake8]
+ignore =
+    E265, # block comment should start with '# '
+    E266, # too many leading '#' for block comment
+    E402, # module level import not at top of file
+    E722, # do not use bare 'except'
```

Comment:
There are also `per-file-ignores` as in: https://github.com/OSGeo/grass/blob/master/lib/python/.flake8#L34 if that would be helpful here to avoid whole another file (well, unless it is two mostly non-overlapping sets).

Reply:
I was simply following the GRASS-core repo

Reply:
The core repo has 4 very different code bases (with different styles and errors), so that's the reason for different .flake8 files there (for now!). It is also using different job definitions there, but my reasons for that are no longer valid (Black or Pylint would now go to a separate file, for example), so I should redo it.